### ISSUE TITLE
Fission CLI using old libssl / broken in Ubuntu 22.04

### ISSUE THREAD

**Issue by tennox** [fission-codes/fission#635]
# Summary

I'm maintaining a [self-upgrading docker image for fission cli](https://gitlab.com/txlab/docker/fission-cli/-/merge_requests/4) :sunglasses:, and the [CI pipeline](https://gitlab.com/txlab/docker/fission-cli/-/jobs/3795087886) for **Ubuntu 20.04 -> 22.04** showed me this:

```
$ fission --help
fission: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```

I checked the installation guide but after some research found [libssl 1.1 was deprecated and is removed in favor of **libssl3**](https://askubuntu.com/a/1412746)

### Impact

The docker image will just stay with ubuntu 20.04 for now, but users with Ubuntu 22 won't easily be able to use your cli.

## Solution

Describe the sort of fix that would solve the issue

# Detail

**To Reproduce**
- Install `https://github.com/fission-suite/fission/releases/download/2.18.0/fission-cli-ubuntu-20.04-x86_64` on Ubuntu 
- Or run: `docker run --rm -it registry.gitlab.com/txlab/docker/fission-cli:b43d947b18aefe3eb276b273b5836d3d00cd3cff`

**Comment by madclaws** [comment]
yes, right now i am using fission-cli with a separate install of openssl 1.1 adding the path in `LD_LIBRARY_PATH`.
